Header header
int8 servo_id
int16 position
int16 speed
int8 volt
int8 temp